src/social.cs

```diff
@@ -58,12 +58,11 @@ interface SLServiceType {
 		[Mac (10,9)]
 		NSString TencentWeibo { get; }
 
-#if MONOMAC
 		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use LinkedIn SDK instead.")]
 		[Field ("SLServiceTypeLinkedIn")]
+		[NoiOS]
```

Comment:
I've been using:

[NoiOS][NoMacCatalyst][NoWatch][NoTV]

to make it clear it is only macOS, though that may be overkill.